### PR TITLE
Disabled multiple button taps

### PR DESCRIPTION
**Pull request by potvik**

### CODE REVIEW

**Comment by theofandrich** [comment]
Other buttons are disabled when Tap to Speak is Pressed

File: voice/voice-ai/x/Actions/ActionsView.swift [hunk 1]
```diff
@@ -273,9 +292,14 @@ struct ActionsView: View {
                 
                 let isPressed: Bool = true
                 
-                GridButton(currentTheme: currentTheme, button: button, foregroundColor: .black, active: isSpeakButtonPressed, isPressed: isPressed) {}.simultaneousGesture(
+                GridButton(currentTheme: currentTheme, button: button, foregroundColor: .black, active: isSpeakButtonPressed, isPressed: isPressed) {event in
+                    self.setLastButtonPressed(action: button.action, event: event)
+                    if((event) != nil) { return }
```
Comment (polymorpher) [inline voice/voice-ai/x/Actions/ActionsView.swift:297]: this line doesn't do anything

File: voice/voice-ai/x/Actions/ActionsView.swift [hunk 2]
```diff
@@ -232,7 +248,9 @@ struct ActionsView: View {
         if button.action == .speak {
             if button.pressedLabel != nil {
                 // Press to Speak & Press to Send
-                GridButton(currentTheme: currentTheme, button: button, foregroundColor: .black, active: self.isTapToSpeakActive, isPressed: self.isTapToSpeakActive, clickCounterStartOn: 100) {
+                GridButton(currentTheme: currentTheme, button: button, foregroundColor: .black, active: self.isTapToSpeakActive, isPressed: self.isTapToSpeakActive, clickCounterStartOn: 100) {event in
+                   if((event) != nil) { return }
```
Comment (polymorpher) [inline voice/voice-ai/x/Actions/ActionsView.swift:252]: What would make `event == nil`?

Comment (polymorpher) [reply voice/voice-ai/x/Actions/ActionsView.swift:252]: nevermind, i found out from the code in GridButton

**Comment by polymorpher** [comment]
It seems a bit convoluted. Wish we can do something simpler to achieve the same goal

**Pull request merged**